pkg/modules/observability.go

```diff
@@ -897,6 +899,19 @@ func PowerMaxMetrics(ctx context.Context, isDeleting bool, op utils.OperatorConf
 		return fmt.Errorf("could not find deployment obj")
 	}
 
+	// Dynamic secret/configMap mounting is only supported in v2.14.0 and above
+	secretSupported, _ := utils.MinVersionCheck("v2.14.0", cr.Spec.Driver.ConfigVersion)
```

Comment:
Any other locations where we perform similar version check and could use a constant variable for the version?